### PR TITLE
Implement alternative icons, fixes #167

### PR DESCRIPTION
**Pull request by hasufell**

### CODE REVIEW

**Comment by nazar-pc** [comment]
This works nicely, thanks!

File: src/ui-tray-icon.c [hunk 1]
```diff
@@ -154,18 +174,26 @@ pixbuf_array_new(int size)
...
-		 */
-		if (pixbufs[VOLUME_OFF] == NULL)
-			pixbufs[VOLUME_OFF] = pixbuf_new_from_stock("audio-volume-low", size);
+		pixbufs[VOLUME_MUTED] = pixbuf_new_from_stock(
+		(const gchar*[]) {
+			"audio-volume-muted-panel", "audio-volume-muted", NULL
+		}, size);
+		pixbufs[VOLUME_OFF] = pixbuf_new_from_stock(
+		(const gchar*[]) {
+			"audio-volume-off-panel", "audio-volume-off", "audio-volume-low-zero-panel", "audio-volume-low-panel", "audio-volume-low", NULL
```
Comment (nazar-pc) [inline src/ui-tray-icon.c]: I'd add `audio-volume-low-zero` after `audio-volume-low-zero-panel` for completeness

File: src/ui-tray-icon.c [hunk 2]
```diff
@@ -87,28 +87,48 @@ pixbuf_new_from_file(const gchar *filename)
 /**
  * Looks up icons based on the currently selected theme.
  *
- * @param icon_name icon name to look up
+ * @param icon_names array of icon names to look up, the first
```
Comment (elboulangero) [inline src/ui-tray-icon.c]: I would explicitly write "null-terminated array"

**Comment by elboulangero** [comment]
@nazar-pc do you have a link that documents this `-panel` thing ? You say it's Ubuntu specific, is it correct ?

**Comment by nazar-pc** [comment]
I don't have docs and not sure it is Ubuntu-specific, might be more common or not.
I've noticed this when tried PNMixer with different themes and found that not all of them result in proper icon, so I've digged deeper to find out the reason and submitted #167.

**Comment by elboulangero** [comment]
Thanks everyone for time and feedback !

**Pull request merged**